tests/test_compute_engines/yield_curve.py

```diff
@@ -84,7 +84,7 @@ async def get_yield_points(output_decimals):
     for on_key in on_keys:
         # fetch data from oracle
         on_value, on_decimals, last_updated_timestamp, _ = await client.get_value(
-            on_key, DEFAULT_AGGREGATION_MODE
+            on_key, BaseConfig.DEFAULT_AGGREGATION_MODE
```

Comment:
This is good, though can be even better - following the update to the contract: https://github.com/42labs/Empiric/commit/b01c8398e1336a003be0bc157126c3532e97f1bc

Reply:
Just added a class for AggregationMode. Would still keep it as a config variable.